pkg/master/master.go

```diff
@@ -84,7 +85,6 @@ import (
 	daemonetcd "k8s.io/kubernetes/pkg/registry/daemonset/etcd"
 	horizontalpodautoscaleretcd "k8s.io/kubernetes/pkg/registry/horizontalpodautoscaler/etcd"
 
-	"github.com/emicklei/go-restful"
```

Comment:
why would this line be deleted in cherry-pick? It's not deleted in the original PR #14929.

Thank you @JanetKuo. I will handle this myself. I may submit another cherry-pick request.